src/components/components/HeroSection/HeroSection.scss

```diff
@@ -0,0 +1,101 @@
+.HeroSection {
+	padding-top: 5%;
+	max-width: 1260px;
+	width: 80%;
+	max-height: 673px;
+	height: 80vh;
+	margin: 0 auto;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	gap: 3%;
+	font-family: var(--inter);
+
+	&__info {
+		padding: 30px 0;
+	}
+
+	&__info,
+	&__image {
+		flex-basis: 50%;
+		max-width: 50%;
+	}
+
+	&__title {
+		margin: 0;
+		font-size: calc(32px + (56 - 32) * ((100vw - 768px) / (1280 - 768)));
+		line-height: 133%;
+		font-weight: 700;
```

Comment:
do we need font weight? works the same without it in mine

Reply:
I guess this is because we imported the fonts incorrectly (in our indexx.scss @import we haven't specify the desired styles) Could it be the case? @jasper-prgh

Reply:
@tinkabel85 No, but I think we should move the font-weight to the global heading-styles